src/util.cpp

```diff
@@ -802,10 +802,11 @@ int GetNumCores()
 std::string CopyrightHolders(const std::string& strPrefix)
 {
     std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
+    if (strCopyrightHolders.find("%s") != std::string::npos) {
         strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
     }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    if (std::string("Bitcoin Core") != COPYRIGHT_HOLDERS_SUBSTITUTION) {
```

Comment:
@sipa, definitely.
Please, don't do what I suggested.
`if (strcmp("Bitcoin Core", COPYRIGHT_HOLDERS_SUBSTITUTION) != 0)` can also be evaluated at compile time, but that's C, not C++ and is harder to read than what the patch currently does. Thus, just forget my suggestion.